src/ai/backend/gateway/vfolder.py

```diff
@@ -213,6 +215,14 @@ async def create(request: web.Request, params: Any) -> web.Response:
              params['usage_mode'].value, params['permission'].value)
     folder_host = params['folder_host']
     unmanaged_path = params['unmanaged_path']
+    # Check if quota is in right format and covert to BinarySize type
+    if params['quota'] is None:
+        quota = None
+    else:
+        try:
+            quota = BinarySize(params['quota'])
+        except Exception:
+            raise InvalidAPIParameters('Quota cannot be converted to BinarySize.')
```

Comment:
It's already validated and converted by `check_api_params()`.
Please checkout and remove the duplicates.